extensions/functions_string.yaml

```diff
@@ -163,3 +165,146 @@ scalar_functions:
           - value: "fixedchar<L1>"
           - value: "varchar<L2>"
         return: "BOOLEAN"
+  -
+    name: ltrim
+    description: >-
+      Remove any occurrence of the characters from the left side of the string.
+      If no characters are specified, spaces are removed.
+    impls:
+      - args:
+          - value: "varchar<L1>"
+          - value: "varchar<L2>"
```

Comment:
Oh yeah, I wasn't aware that I could do this.  I'll make these changes.  I came across a few other functions that probably could be more descriptive like this as well.  I can submit a separate PR for those later.

Reply:
Would it make more sense to do something like this directly in the function description instead of repeating it for all implementations of the function?

```
    name: ltrim
    description: >-
      Remove any occurrence of the characters from the left side of the string.
      If no characters are specified, spaces are removed.

      arg0: input - The string to remove characters from.
      arg1: characters - The set of characters to remove.```